gammapy/scripts/spectrum_pipe.py

```diff
@@ -0,0 +1,220 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (print_function)
+import logging
+from ..utils.scripts import get_parser, set_up_logging_from_args
+from ..obs import DataStore, ObservationTable
+from ..irf import EnergyDispersion, EnergyDispersion2D
+from ..irf import EffectiveAreaTable, EffectiveAreaTable2D
+from ..data import CountsSpectrum, EventList
+from ..spectrum import EnergyBounds, Energy
+from astropy.coordinates import Angle, SkyCoord
+import sherpa.astro.ui as sau
```

Comment:
The Sherpa imports have to be moved inside the functions / methods, because it's an optional dependency:
https://travis-ci.org/gammapy/gammapy/jobs/82537346#L973